src/app/app-routing.module.ts

```diff
@@ -5,6 +5,7 @@ import { ResourcesComponent } from './components/resources/resources.component';
 import { LanguagesComponent } from './components/languages/languages.component';
 import { AttachmentsComponent } from './components/attachments/attachments.component';
 import { AuthGuardService } from './service/auth-guard/auth-guard.service';
+import { OklaLoginComponent } from './components/okla-login/okla-login.component';
```

Comment:
Everywhere it says `Okla` it should say `Okta`

Reply:
Fixed. Sorry for the typo.